pytorch_blade/src/compiler/mlir/converters/torch_mlir_op_filter.cpp

```diff
@@ -107,13 +109,13 @@ const std::unordered_set<std::string> &GetTorchMlirWhiteList() {
       std::istringstream f(custom_ops);
       std::string s;
       while (getline(f, s, ';')) {
-          white_list.insert(s);
+          white_list2.insert(s);
           ostr << s << ", ";
       }
       ostr << "]";
       LOG(INFO) << ostr.str();
   });
-  return white_list;
+  return white_list2;
```

Comment:
Confused about this change, `white_list` is never used?

Reply:
It's just used to debug, I will remove it later.